Formula/flaresolverr.rb

```diff
@@ -0,0 +1,103 @@
+class Flaresolverr < Formula
+  require "language/node"
+
+  desc "Proxy server to bypass Cloudflare protection"
+  homepage "https://github.com/FlareSolverr/FlareSolverr"
+  url "https://github.com/FlareSolverr/FlareSolverr/archive/refs/tags/v2.2.2.tar.gz"
+  sha256 "ffa2a7ec52f90223c33cbefb938d522381eae4fe2cd5ad5087e264093078a532"
+  license "MIT"
+  head "https://github.com/FlareSolverr/FlareSolverr.git", branch: "master"
+
+  depends_on "node@16"
+
+  def node
+    deps.reject(&:build?)
+        .map(&:to_formula)
+        .find { |f| f.name.match?(/^node(@\d+(\.\d+)*)?$/) }
+  end
+
+  def install
+    libexec.install Dir["*"]
+
+    cd libexec do
+      ENV["PUPPETEER_SKIP_DOWNLOAD"] = "1"
+      system "npm", "install", *Language::Node.local_npm_install_args
+      system "npm", "run", "build"
+    end
+
+    puppeteer_download_path = var/"flaresolverr/puppeteer"
+    puppeteer_download_path.mkpath
+
+    (bin/"flaresolverr").write <<~EOS
+      #!/bin/bash
+      DEFAULT_PUPPETEER_EXECUTABLE_PATH=(${PUPPETEER_DOWNLOAD_PATH:-"#{puppeteer_download_path}"}/*/Firefox*.app/Contents/MacOS/firefox)
+      PUPPETEER_EXECUTABLE_PATH="${PUPPETEER_EXECUTABLE_PATH:-$DEFAULT_PUPPETEER_EXECUTABLE_PATH}" PUPPETEER_PRODUCT="${PUPPETEER_PRODUCT:-firefox}" exec "#{node.opt_bin}/node" "#{libexec}/dist/server.js" "$@"
+    EOS
+
+    (bin/"flaresolverr-install-browser").write <<~EOS
+      #!/bin/bash
+
+      cd "#{libexec}" || exit
+
+      OPTIND=1
+      while getopts "hr" opt; do
+        case "$opt" in
+          h)
+            echo "usage: $(basename $0) [-h] [-r]"
+            echo "  -h    show help"
+            echo "  -r    remove existing browser and reinstall"
+            exit 0
+            ;;
+          r)
+            rm -rf "#{puppeteer_download_path}" || exit
```

Comment:
Shouldn't this remove whatever the variable is set to?

Reply:
Yes, good point. Forgot to change this.